src/application/dashboard/resolvers.ts

```diff
@@ -2,23 +2,28 @@ import { DashboardService } from './service';
 import Submission from '../../domain/submission/services/models/submission';
 import { SubmissionId } from '../../domain/submission/types';
 import { IResolvers } from 'apollo-server-express';
+import { UserService } from 'src/domain/user';
 
-const resolvers = (dashboard: DashboardService): IResolvers => ({
+const resolvers = (dashboard: DashboardService, userService: UserService): IResolvers => ({
     Query: {
-        async getSubmissions(): Promise<Submission[] | null> {
-            return await dashboard.find();
+        async getSubmissions(_, {}: {}, context): Promise<Submission[] | null> {
+            const user = await userService.getCurrentUser(context.authorizationHeader);
```

Comment:
Quick observation. This isn't standard with how apollo a decode to userId context. Plus, I noticed this header is passed to a repo.. not sure the repo needs to know what a HTTP header is when we have the user id.

https://www.apollographql.com/docs/apollo-server/security/authentication/

Reply:
I just noticed Erez made the same comment in a way.

Reply:
Responded to Erez